development/build/manifest.js

```diff
@@ -11,6 +11,7 @@ const baradDurManifest = isManifestV3
   ? require('../../app/manifest/v3/_barad_dur.json')
   : require('../../app/manifest/v2/_barad_dur.json');
 const { loadBuildTypesConfig } = require('../lib/build-type');
+const manifestFlags = require('../../manifest-flags.json');
```

Comment:
2. Read customized manifestFlags from `manifest-flags.json` so we can override the one from controller (API) - MV3 build